src/components/navBar/styled.ts

```diff
@@ -0,0 +1,71 @@
+import { NavLink } from 'react-router-dom';
+import { Box, Divider, Typography, styled } from '@mui/material';
+import { colors, fonts } from '../../theme';
+
+const NavBarContainer = styled(Box)`
```

Comment:
Please find an information which tags you need to use for navigation in side bar

Reply:
it shouldn't be just i div in div

Reply:
fix: 'nav' tag instead of div, links wrapped in 'ul' tag mapping 'li'.

Upd: I've read about <section> tag for navigations, but since it is major navigation we do not this additional tag, right?